src/test/java/seedu/address/logic/parser/DeleteGoodCommandParserTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_SUPPLIER;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.DeleteGoodCommand;
+
+/**
```

Comment:
I am not sure whether this is supposed to be in the class comment?

Reply:
That was the adopted explanation from AB3. See DeleteSupplierCommandParserTest.  
I'm not sure where would be better to put this para if not at the head of the test class.